samples/bootloader/sysbuild/mcuboot_s1_image.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		zephyr,code-partition = &boot_partition_slot1;
```

Comment:
boot partition slot1 is not named right, <x>_partition where x is something indicating s1/b0 and not using the slot1 MCUboot name

Reply:
Agree, at first I've tried to allow that the board variant should support NSIB both with and without MCUBOOT, but this causes more issues than it has benefits, it makes more sense to have separate variants for that. See now.